controllers/api/v3.js

```diff
@@ -2402,6 +2407,114 @@ apiController.getGenieDeviceCollection = async function(request, response) {
 };
 
 
+/**
+ * Get all the mesh repeaters from a device. Given the mac address of a device
+ * (mesh master).
+ *
+ * @memberof controllers/api/v3
+ *
+ * @async
+ *
+ * @param {HTTPRequest} request - The HTTP request.
+ * @param {HTTPResponse} response  - The HTTP response.
+ */
+apiController.getMeshRepeaters = async function(request, response) {
+  // Validate the request
+  let validation = apiController.validateRequest(request);
+
+  if (!validation.valid) {
+    return response.status(validation.statusCode).json({
+      success: validation.valid,
+      message: validation.message,
+      repeaters: [],
+    });
+  }
+
+  // Validate the MAC
+  validation = apiController.validateField(request.params, 'mac');
+
+  if (!validation.valid) {
+    return response.status(validation.statusCode).json({
+      success: validation.valid,
+      message: validation.message,
+      repeaters: [],
+    });
+  }
+
+  // Search through all repeaters with the mesh master
+  let repeaters = null;
+  let device = null;
+
+  // Try finding both
+  try {
+    // Search through repeaters and device at the same time, find the device to
+    // check if the device at least exists in database and has the mesh mode
+    // vendor enabled
+    let repeaterPromise = RepeaterModel.find(
+      {mesh_master: request.params['mac']},
+    );
+    let devicePromise = DeviceModel.findOne(
+      {_id: request.params['mac']}, {_id: true, mesh_mode: true},
+    );
+
+    [repeaters, device] = await Promise.all(
+      [repeaterPromise.lean(), devicePromise.lean()],
+    );
```

Comment:
```suggestion
    const masterId = request.params['mac'];
    [repeaters, device] = await Promise.all([
      RepeaterModel.find({mesh_master: masterId}).lean(),
      DeviceModel.findById(masterId, {mesh_mode: true}).lean(),
    ]);
```
- Projections sempre incluem o `_id` por default, pode deixar lá por clareza, só vai usar espaço
- Ter as queries dentro do `Promise.all` pra mim faz mais sentido, mas se for pra deixar fora, o `lean` tem que ir junto - não faz sentido separa-los
- `findById` vs `findOne` economiza um bom espaço também